### PR TITLE
Apps 5982 schema tags used for bazaarvoice pbc

### PR DESCRIPTION
**Pull request by TimurSultanov**
## PR Description

Ticket: https://spryker.atlassian.net/browse/APPS-5982

### CODE REVIEW

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 1]
```diff
@@ -66,5 +66,90 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
...
+|----------------------------|----------------------|
+| organization.identifier    | productId            |
+| organization.name          | productName          |
+| organization.logo          | productDescription   |
+
+#### Tracking Pixel
+Core Templates:
+* `SprykerShop/Yves/PaymentPage/Theme/default/views/payment-success/index.twig`
+* `SprykerShop/Yves/CheckoutPage/Theme/default/views/order-success/order-success.twig`
+* `SprykerShop/Yves/MerchantWidget/Theme/default/views/merchant-meta-schema/merchant-meta-schema.twig` (only for a Marketplace)
```
Comment (k4emic) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:105]:
```suggestion
* `SprykerShop/Yves/MerchantWidget/Theme/default/views/merchant-meta-schema/merchant-meta-schema.twig` (only for Marketplace)
```

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 2]
```diff
@@ -66,5 +66,90 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
...
+
+| schema.org property                        | Bazaarvoice property | Only for a Marketplace |
```
Comment (k4emic) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:107]:
```suggestion
| schema.org property                        | Bazaarvoice property | Only for Marketplace |
```

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 3]
```diff
@@ -66,5 +66,90 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
...
+* `SprykerShop/Yves/ShopUi/Theme/default/components/molecules/product-item/product-item.twig`
+* `SprykerShop/Yves/ProductReviewWidget/Theme/default/views/product-review-display/product-review-display.twig`
+* `SprykerShop/Yves/ProductReviewWidget/Theme/default/components/molecules/rating-selector/rating-selector.twig`
+
+| schema.org property         | Class name      |
+|-----------------------------|-----------------|
+| product.sku                 |                 |
+| aggregateRating.ratingValue |                 |
+| aggregateRating.bestRating  |                 |
+|                             | rating-selector |
```
Comment (k4emic) [inline docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:152]: How should these tables be read? I don't understand the meaning of these tables.

**Review by k4emic — CHANGES_REQUESTED** [review]
Needs some adjustments, mostly for the last three tables.
It should be mentioned that all schema.org data has to be embedded using `meta` tags, since we do not support inline data.

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 4]
```diff
@@ -66,5 +66,90 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
 $config[KernelConstants::DOMAIN_WHITELIST][] = '*.bazaarvoice.com';
 ```
 
+### 3. Add markup to custom templates
+
+The Bazaarvoice PBC takes data of products from Yves pages (for example Product Detail Page).
+To get necessary data from the pages are used schemas from [Schema.org](https://schema.org/).
+By default, all the necessary markups are already added to Yves templates.
+
+To custom templates the markups required for the Bazaarvoice PBC must be added according to the tables below.
```
[the remark this answers is not in the document]
Comment (TimurSultanov) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:75]: which one? there are 3 sentences

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md [hunk 5]
```diff
@@ -66,5 +66,90 @@ Alternatively, you may add the domain to the allowlist from the `config/Shared/c
...
+Core Template: `SprykerShop/Yves/ProductDetailPage/Theme/default/views/pdp/pdp.twig`
+
+| schema.org property          | Bazaarvoice property |
+|------------------------------|----------------------|
+| product.sku                  | productId            |
+| product.name                 | productName          |
+| product.description          | productDescription   |
+| product.image                | productImageURL      |
+| product.url                  | productPageURL       |
+| brand.name                   | brandId, brandName   |
```
[the remark this answers is not in the document]
Comment (TimurSultanov) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:87]: exactly, missed this

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md (hunk 3 above)
Comment (TimurSultanov) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:152]: just a list of properties and classes that should be added, how would you suggest describing this?

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (k4emic) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:75]: Sorry, the final sentence:
> To custom templates the markups required for the Bazaarvoice PBC must be added according to the tables below.

**Comment by TimurSultanov** [comment]
> Needs some adjustments, mostly for the last three tables. It should be mentioned that all schema.org data has to be embedded using `meta` tags, since we do not support inline data.

we don't check tags actually

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (k4emic) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:75]: "If you have custom templates or are making your own frontend..."

File: docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md (hunk 3 above)
Comment (k4emic) [reply docs/pbc/all/ratings-reviews/third-party-integrations/integrate-bazaarvoice.md:152]: Summary: We will need to provide examples of what HTML markup must be present for customers who are making their own frontend (or significant changes to the default spryker frontend).

**Review by k4emic — APPROVED** [review]

**Review by k4emic — DISMISSED** [review]

**Pull request merged**